editor/plugins/script_text_editor.cpp

```diff
@@ -506,6 +480,40 @@ void ScriptTextEditor::_validate_script() {
 		errors_panel->pop(); // Cell.
 	}
 	errors_panel->pop(); // Table
+}
+
+void ScriptTextEditor::_validate_script() {
+	CodeEdit *te = code_editor->get_text_editor();
+
+	String text = te->get_text();
+	List<String> fnc;
+	Set<int> safe_lines;
+
+	warnings.clear();
+	errors.clear();
+
+	if (!script->get_language()->validate(text, script->get_path(), &fnc, &errors, &warnings, &safe_lines)) {
+		String error_text = TTR("Error at ") + "(" + itos(errors[0].line) + "," + itos(errors[0].column) + "): " + errors[0].message;
```

Comment:
I know you just moved this code but since I notice it now, would be good to fix. This is bad `TTR` usage for translation as it includes only part of a sentence, so many languages wouldn't be able to provide a valid translation.

This should be something like this:
```suggestion
		// TRANSLATORS: Script error pointing to a line and column number.
		String error_text = vformat(TTR("Error at (%d, %d):"), errors[0].line, errors[0].column) + " " + errors[0].message;
```

Reply:
Done